src/plots/cartesian/graph_interact.js

```diff
@@ -23,6 +23,10 @@ var dragElement = require('../../components/dragelement');
 
 var fx = module.exports = {};
 
+// TODO remove this in version 2.0
+// copy on Fx for backward compatible
+fx.unhover = dragElement.unhover;
```

Comment:
@alexcjohnson fyi:

![image](https://cloud.githubusercontent.com/assets/6675409/14475694/49759c40-00d1-11e6-9606-0f215e625d2b.png)

more info in: https://github.com/plotly/plotly.js/pull/390#discussion_r58339588